src/components/QrScanner/QrScanner.tsx

```diff
@@ -197,20 +172,30 @@ const QrCheckIn: React.FC<QrProps> = ({
   };
 
   const checkInUser = (id: string, fname: string) => {
+    let year;
```

Comment:
relevant changes here, everything else is prettier reformatting to style of rest of the codebase